lib/Fmt_odoc.ml

```diff
@@ -13,7 +13,7 @@ open Fmt
 open Odoc_parser.Ast
 module Loc = Odoc_parser.Loc
 
-type conf = {fmt_code: Fmt.code_formatter}
+type conf = {fmt_code: Fmt.code_formatter; normalize_spaces: bool}
```

Comment:
What `normalize_spaces` means ? From looking at the implementation, it's making sure verbatim blocks have breaks. Why is this needed ? What about code and math blocks ?